pkgs/applications/editors/oni2/common.nix

```diff
@@ -0,0 +1,252 @@
+{ lib, stdenv, nodePackages
+# Fetch dependencies
+, fetchFromGitHub, gitMinimal, curlMinimal, cacert, yarn, unzip, xorg, nodejs
+, ripgrep, fontconfig, libGL, libGLU, ncurses, acl, harfbuzz, libjpeg, expat
+, icu58, libpng
+# Build
+, jq, perl, makeWrapper, bash, which, nasm, python, gn, ninja, cmake, clang
```

Comment:
Can we use python3?

Reply:
No, we can't. Fetcher for `esy_skia` requires python2.